pyproject.toml

```diff
@@ -12,16 +12,157 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "nemo_curator"
+description = "Scalable Data Preprocessing Tool for Training Large Language Models"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "Joseph Jennings", email = "jjennings@nvidia.com" },
+    { name = "Mostofa Patwary", email = "mpatwary@nvidia.com" },
+    { name = "Sandeep Subramanian", email = "sasubramania@nvidia.com" },
+    { name = "Shrimai Prabhumoye", email = "sprabhumoye@nvidia.com" },
+    { name = "Ayush Dattagupta", email = "adattagupta@nvidia.com" },
+    { name = "Vibhu Jawa", email = "vjawa@nvidia.com" },
+    { name = "Jiwei Liu", email = "jiweil@nvidia.com" },
+    { name = "Ryan Wolf", email = "rywolf@nvidia.com" },
+    { name = "Sarah Yurick", email = "syurick@nvidia.com" },
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+]
+requires-python = ">=3.10, <3.11"
+dependencies = [
```

Comment:
According to [setuptools documentation](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#dynamic-metadata), I could dynamically link `dependencies` and `optional-dependencies` to requirements files, but this is currently in beta. Instead, I just list them all in here and delete the txt files.

Happy to hear other preferences.

Reply:
fine with me

Reply:
I'm not aware of any reason to prefer the requirements files unless you specifically wanted to distribute them.